stdlib/toolchain/legacy_layouts/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 add_custom_target("copy-legacy-layouts" ALL)
 
 foreach(sdk ${SWIFT_SDKS})
-  foreach(arch ${SWIFT_SDK_${sdk}_ARCHITECTURES})
+  foreach(arch ${SWIFT_SDK_${sdk}_ARCHITECTURES} ${SWIFT_SDK_${sdk}_MODULE_ARCHITECTURES})
```

Comment:
If these are overlapping this might run an architecture multiple times, correct?
Is this fine because this is only copying files & adding targets that copy?

Reply:
Those should be no overlapping architecture, since we check for duplicates in `${SWIFT_SDK_${sdk}_MODULE_ARCHITECTURES}` when including `SwiftConfigureSDK.cmake` in the main `CMakeLists.txt`

https://github.com/apple/swift/blob/main/cmake/modules/SwiftConfigureSDK.cmake#L212

(`list_subtract` is defined at https://github.com/apple/swift/blob/main/cmake/modules/SwiftList.cmake#L3)

If that were not the case, we would need to filter duplicates, otherwise CMake would throw an error when adding a target with the same name as an existing one (at least in my experience).